platform/ios/app/MBXUserLocationAnnotationView.m

```diff
@@ -9,46 +10,168 @@ @interface MBXUserLocationAnnotationView()
 
 @implementation MBXUserLocationAnnotationView
 
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
```

Comment:
Nit: The structure of this implementation works, but we should adhere to the format used in the initializer code snippet. Consistency with initializers makes it easier for us to spot errors like missing assignments to self or return statements.

```objc
if (self = [super initWithFrame:frame]) {
    self.backgroundColor = [UIColor clearColor];
}
return self;
```